src/test/routes/notice-end.test.ts

```diff
@@ -12,7 +12,7 @@ describe(`GET ${PageUrls.NOTICE_END}`, () => {
 });
 
 describe(`on POST ${PageUrls.NOTICE_END}`, () => {
-  test('should navigate to the notice pay page when save and continue button is clicked', async () => {
+  test('should navigate to the notice end page when save and continue button is clicked', async () => {
```

Comment:
Test text was correct previously, 'should navigate to the notice **pay** page when save and continue button is clicked'